daemon/k8s_watcher.go

```diff
@@ -1809,16 +1811,19 @@ func cnpNodeStatusController(ciliumV2Store cache.Store, cnp *cilium_v2.CiliumNet
 			break
 		}
 
-		// Wait a small amount of time to try to update again.
-		logger.WithError(cnpUpdateErr).Warningf("update of CNP failed; sleeping for %s amount of time before trying again", updateWaitDuration)
+		logger.WithError(cnpUpdateErr).Debugf("Update of CNP status failed. Sleeping for %s before retrying", updateWaitDuration)
 		time.Sleep(updateWaitDuration)
 	}
 
 	if cnpUpdateErr != nil {
-		return cnpUpdateErr
+		overallErr = cnpUpdateErr
 	} else {
-		return waitForEPsErr
+		overallErr = waitForEPsErr
 	}
+
+	logger.WithError(overallErr).Warningf("Update of CNP status failed %s times. Will keep retrying.", maxAttempts)
```

Comment:
Won't this always be printed regardless `overallErr` is or isn't nil? Also, `failed %s times` -> `failed %d times`

Reply:
🤦‍♀️ Fixed